### PR TITLE
[RPD-172] Improve documentation

### PR DESCRIPTION
**Pull request by archena**
Various updates to readmes and documentation site.

## Checklist

Please ensure you have done the following:

* [ ] I have read the [CONTRIBUTING](https://github.com/fuzzylabs/matcha/blob/main/CONTRIBUTING.md) guide.
* [ ] I have updated the documentation if required.
* [ ] I have added tests which cover my changes.

## Type of change

Tick all those that apply:

* [ ] Bug Fix (non-breaking change, fixing an issue)
* [ ] New feature (non-breaking change to add functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Other (add details above)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,38 +18,40 @@
...
+If you train machine learning models, then you know the challenge of going from _experiment_ to _production_. There's a vast range of tools that promise to help, from experiment tracking through to model deployment, but setting these up requires a lot of time and cloud engineering knowledge.
 
-With a single command and using sensible defaults, Data Scientists and ML practitioners are provided with the following capabilities:
-
-* &#127939; A way to run model training.
-* &#128099; A way to track experiments.
-* &#127869;&#65039; A way to deploy and serve models.
-
-&#127861; `matcha` uses open source tools to build this MLOps environment on Azure.
+**Matcha removes the complexity of provisioning your machine learning infrastructure**. With one step, you'll have a complete _machine learning operations (MLOPs)_ stack up and running in your Microsoft® Azure cloud environment. This means you'll be able to track your experiments, train your models, as well as deploy and serve those models.
```
Comment (tomstockton) [inline README.md:26]: s/MLOPs/MLOps/

File: README.md [hunk 2]
```diff
@@ -18,38 +18,40 @@
...
-Keen to get started with `matcha`? We have a great getting started guide which you can find [here](https://mymatcha.ai/getting-started/).
+# &#127939; How do I get started?
 
-By the end, you'll understand how to use `matcha` and have a fully provisioned movie recommendation system in Azure, ready to serve some movie ratings!
+If you're new to Matcha, the best place to start is [our guide to deploying your first model](http://www.mymatcha.ai/getting-started). If you're happy with the basics, then you might want to dive into our [Matcha examples](https://github.com/fuzzylabs/matcha-examples) on Github.
 
 ## &#128214; Documentation
 
-Want to learn more about `matcha`? See our [documentation](https://mymatcha.ai/) which covers what `matcha` is in more detail, how to use it, what Azure permissions are required, and much more!
+The full `matcha` documentation can be found at [mymatcha.ai](https://mymatcha.ai). This covers what `matcha` is in more detail, how to use it, what Azure permissions are required, and how Matcha works internally.
```
Comment (JonoCX) [inline README.md:42]: There's a mix of Matcha and `matcha`, we should settle on one or the other

File: docs/index.md [hunk 3]
```diff
@@ -1,36 +1,45 @@
-`matcha` is a tool for provisioning MLOps environments to the cloud.
+# Introduction
 
-Jump to [get started](https://fuzzylabs.github.io/matcha/getting-started/) section to help you get started using matcha.
+If you train machine learning models, then you know the challenge of going from _experiment_ to _production_. There's a vast range of tools that promise to help, from experiment tracking through to model deployment, but setting these up requires a lot of time and cloud engineering knowledge.
 
-## Why do you need `matcha`?
+**Matcha removes the complexity of provisioning your machine learning infrastructure**. With one step, you'll have a complete _machine learning operations (MLOPs)_ stack up and running in your Microsoft® Azure cloud environment. This means you'll be able to track your experiments, train your models, as well as deploy and serve those models.
```
Comment (tomstockton) [inline docs/index.md:5]: s/MLOPs/MLOps/

File: docs/getting-started.md [hunk 4]
```diff
@@ -1,82 +1,78 @@
...
-
-```bash
 python3 -m venv venv
 source venv/bin/activate
 ```
 
-> There is a requirement for the Python version being used in this example to be 3.8+. We recommended making use of [`pyenv`](https://github.com/pyenv/pyenv) to manage your versions.
-
-Install `matcha`:
+Now, let's install `matcha`:
```
Comment (JonoCX) [inline docs/getting-started.md:39]: Similar mix of Matcha/`matcha` in here too

File: docs/index.md (hunk 3 above)
Comment (JonoCX) [inline docs/index.md:5]: MLOPs -> MLOps

File: docs/getting-started.md [hunk 5]
```diff
@@ -136,23 +132,23 @@ Experiment tracker
...
 
-### Running your recommender
+# Training a model
 
-The environment is provisioned, you've got a movie recommender, and you're hyped and ready to go - we hope.
+To train the model, you'll need to do two things
 
-Running the following command will run the recommendation pipeline locally (using ZenML), but the metadata associated with it (e.g., the RMSE performance metric) will be stored in your deployed experiment tracker:
+```bash
+./setup.sh
```
Comment (tomstockton) [inline docs/getting-started.md:140]: reasons for this are vague, what does it do?

File: docs/index.md [hunk 6]
```diff
@@ -1,36 +1,45 @@
...
-* A way to run model training.
-* A way to track experiments.
-* A way to deploy and serve models.
+If you're happy with the basics, then you might want to dive into our [Matcha examples](https://github.com/fuzzylabs/matcha-examples) on Github. More advanced users may be interested in learning [how Matcha works](inside-matcha.md) inside.
 
-The current version of `matcha` provisions the above with the single command, using sensible defaults for the infrastructure.
+# Why use Matcha?
 
-There are more components that could be added here, for example, managing and versioning datasets, the management of models (registration and governance), and monitoring models. Adding all of these is on our roadmap.
+These days there are lots of MLOPs platforms and tools to choose from, and you might wonder what's different about Matcha.
```
Comment (JonoCX) [inline docs/index.md:25]: MLOPs -> MLOps

File: mkdocs.yml [hunk 7]
```diff
@@ -1,26 +1,25 @@
-site_name: matcha-ml
-site_description: matcha-ml is your one-stop-shop for efficiently provisioning MLOps tooling.
+site_name: "Open source MLOPs on Azure in one step"
```
Comment (JonoCX) [inline mkdocs.yml:1]: MLOPs -> MLOps

File: mkdocs.yml [hunk 8]
```diff
@@ -1,26 +1,25 @@
...
+site_description: provision your MLOPs stack to Azure in one step
```
Comment (JonoCX) [inline mkdocs.yml:2]: MLOPs -> MLOps

File: docs/index.md [hunk 9]
```diff
@@ -1,36 +1,45 @@
...
 
-One alternative to this would be to build your own MLOps environment by selecting and combining individual open-source tools. Typically, each of these tools are good at one specific thing, though building a complete pipeline can be complex and time consuming. Broadly, there are a few components which you'll need for an MLOps solution:
+Along with setting up your infrastructure, Matcha comes with a set of well-engineered examples, covering common machine learning use-cases. These examples will help you make the most out of Matcha.
 
-* An environment in which to do Machine Learning efficiently.
-* The ability to hook that environment up to your ML workflow.
-* Knowledge around what best practise looks like for MLOps.
+![Diagram illustrating how Matcha provisions a stack of MLOPs tools on top of Kubernetes](img/stack-diagram.png)
```
Comment (JonoCX) [inline docs/index.md:11]: MLOPs -> MLOps

**Pull request merged**